File: November/Day 3/Javascript/fizz buzz.js
function myFunc(n){
    // variable use to track value of n
    let i = 1;
    while(i <= n){
        // check if i is a multiple of 3 and 5
        if(i % 3 == 0 && i % 5 == 0)
        console.log('fizzbuzz');
        // check if i is a multiple of 3
        else if(i % 3 == 0)
        console.log('fizz');
        // check if i is a multiple of 5
        else if(i % 5 == 0)
        console.log('buzz');
        // else we print the values of i and increment and check conditions again
        else
        console.log(i);
        i++;
    }
    }    
    // passing values to the function by asking input from user
myFunc(prompt('Enter a number'));

File: November/Day 29/C/Fizz-Buzz.c
#include <stdio.h>

void fizzBuzz(int n){
    for(int i = 1; i <= n; i++){
       
       if(i>=5 && i%5==0 && i >= 3 && i%3 == 0){
           printf("fizzbuzz\n");
        }
        else if(i >= 3 && i%3 == 0){

            printf("fizz\n");

        }else if(i>=5 && i%5==0) {
            printf("buzz\n");
        }
        
        else{
            printf("%d\n", i);
        }
        
    }
    return;
}

int main() {

fizzBuzz(30);
}